### DISCUSSION TITLE
Pictures suggestions

### DISCUSSION THREAD

**Discussion by GreatSymphonia** [sawyerpollard/Minecraft-Weather-New-Tab#14] [Show and tell]
In issue #4 , it was suggested to add images sources, we could add our pictures (4k min, 8k preferred) and our sources here.

:)

**Comment by vulpesfelix** [comment]
4k minimum? Minecraft screenshots are 1920x1080 by default...

**Reply by GreatSymphonia** [reply]
Using optifine plus custom shaders/mods, it is possible to get 4k/8k screenshots.

**Comment by vulpesfelix** [comment]
Found these:
https://whvn.cc/kwvrv7
https://whvn.cc/r25w5q
https://whvn.cc/rd7lv1
https://whvn.cc/42rv7m
